one-pager.md

```diff
@@ -1,33 +1,47 @@
 # Pioneer Valley Housing Co-op Seeks Property 
 
-Are you looking to do something unique with your land that fosters community, equity, sustainability, and stewardship for generations to come? We are a group of renters who live in the Pioneer Valley who have joined together as a group equity housing co-op. We are creating a local alternative to traditional homeownership where we can collectively invest in ourselves, our community, and the environment. 
+Are you interested in housing that fosters community, equity, sustainability, and stewardship for generations to come? What about a local alternative to renting and traditional homeownership? We are forming a group equity housing co-op to collectively invest in ourselves, our community, and the environment.
 
-We are looking for access to land either by donation, agreement or sale with the possibility of the landowner continuing occupancy. We would love to connect with like-minded people and organizations with the following values:
+ We aim to support community and resident wellbeing through being democratic and affordable through our structure as a group equity housing co-op. 
+ 
+**Community** 
+* Multiple homes clustered around common spaces and shared resources  
+* Room for creative projects and enterprises
+* Public access spaces and events
 
-**Community**: We believe that resident wellbeing and community wellbeing is interdependent.
-* Shared common house with multiple dwelling units for privacy
-* Stability of homeownership with the flexible commitment of renting
-* Public communal spaces including public orchards and gardens with [Help Yourself](http://commongreen.weebly.com/)
-* Supportive of local enterprises and creative projects
-* Collective decision making and conflict mediation with [Sociocracy For All](https://www.sociocracyforall.org/)
+**Resident Wellbeing**
+* Flexible commitment of renting
+* Security and control of traditional homeownership
+* Community of cohousing or ecovillage without financial inaccessibility or cultural conformity/ too intentional? 
 
-**Equity**: We believe housing is a right and should be affordable and democratically governed.
+**Democratic** 
 * Housing co-op will own assets as a group, with residents as equal voting members
+* Collective decision making and conflict mediation
+
+**Affordable** 
 * [Group equity](https://www.nasco.coop/development/handbook/equity) model will ensure low move in share, to be returned when they move out
 * Initial funds will be raised from grants and community investors, not residents
-* Sliding scale monthly costs will be reinvested in the property and used to pay off investors
-* We are working with [Cooperative Development Institute](https://cdi.coop/) to legally incorporate
+* Residents will pay monthly costs to repay community investors
```

Comment:
I think this one could even start with a dumber down line, like... "Residents pay months costs to live there with low upfront costs, making it accessible to people who cannot afford a traditional home"?